test/apps/iodemo/io_demo.cc

```diff
@@ -867,6 +938,7 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
             std::cout << "  -b <number of buffers>     Number of IO buffers to use for communications" << std::endl;
             std::cout << "  -i <iterations-count>      Number of iterations to run communication" << std::endl;
             std::cout << "  -w <window-size>           Number of outstanding requests" << std::endl;
+            std::cout << "  -k <chunk-size>            Size of per data chunk" << std::endl;
```

Comment:
`"  -k <chunk-size>    Split the data transfer to chunks of this size"`

Reply:
Fixed